solutions/cses-1132.mdx

```diff
@@ -80,7 +83,98 @@ int main() {
 	}
 	return 0;
 }
+```				 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java exceeds the time limit on two test cases.
+</Warning>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class TreeDistancesI {
+	// source -> https://cses.fi/problemset/task/1132/
+
+	static ArrayList<Integer>[] adj;
+	static int [] d1 = new int[200001];  // to store first-max length.
+	static int [] d2 = new int[200001]; // to store second-max length.
+	static int [] c = new int[200001]; // to store child for path of max length.
+
+	public static void dfs(int v, int p) {
```

Comment:
Comments explaining what each dfs method does would be nice

Reply:
agreed

Reply:
what why was this resolved lol

Reply:
i'm not sure what you're seeing but when I view the code there's comments on top of each dfs function describing what it does